aten/src/ATen/core/TensorOptions.h

```diff
@@ -614,4 +614,10 @@ inline DeviceType computeDeviceType(TensorTypeId tid) {
   }
 }
 
+} // namespace at
+
+namespace c10 {
+
+using TensorOptions = at::TensorOptions;
```

Comment:
Let's instead not change the namespace and keep TensorOptions in the c10:: namespace. Everything in the c10 namespace is automatically available in aten:: as well.